package.json

```diff
@@ -1,13 +1,16 @@
 {
   "license": "MIT",
   "scripts": {
-    "start": "node app.js",
+    "start": "npm-run-all --parallel watch:app watch:scss",
+    "watch:app": "nodemon -x 'npm run build:translate && node app.js' --ignore 'apps/**/translations/en/'",
```

Comment:
so this is interesting, it works when I have `start:dev` rather than just `dev`. If I just have `dev` and run `npm run dev` it keeps restarting the server.  Do you know why that is?